### PR TITLE
2023 Steering Election setup

### PR DESCRIPTION
**Pull request by bridgetkromhout**
This PR sets up the 2023 Steering Election. (In the course of setting up 2023, I also corrected issues with the templates and fixed broken links in the docs.)

### CODE REVIEW

**Comment by bridgetkromhout** [comment]
Scheduling notes: consideration was made for avoiding requiring action on holidays such as Labor Day (US), Rosh Hashanah, and Yom Kippur; I also checked the schedule against the likely timeline for K8s 1.29 and I think this schedule will have us finishing before code freeze.

File: elections/steering/2023/README.md
Comment (cblecker) [inline elections/steering/2023/README.md:70]: I think the year might be wrong here

Comment (bridgetkromhout) [reply elections/steering/2023/README.md]: Thanks. Upon review, I found that there were a number of additional places I needed to correct the year.

File: elections/steering/2023/election.yaml
Comment (bridgetkromhout) [inline elections/steering/2023/election.yaml:3]: I _think_ that this field means that I can rename the file from election-template.yaml to election.yaml now, even though the voters.yaml isn't ready yet, because in theory the election doesn't go "live" until this start_datetime.

**Review by jberkus — CHANGES_REQUESTED** [review]
Looks good, one minor change request

File: elections/steering/2023/election.yaml
Comment (bridgetkromhout) [reply elections/steering/2023/election.yaml:3]: > BTW, the election will show in elekto as soon as you merge. Just nobody will be able to vote, and there will be no candidates.

Ohhhh, okay. So possibly I don't want to move the template file to its "live file" name yet? I don't want to create confusion.

Comment (bridgetkromhout) [reply elections/steering/2023/election.yaml]: @jberkus I have tried changing that file back to its template name, so as to avoid Early Empty Election Launch. Perhaps that resolves the immediate issue?

Comment (bridgetkromhout) [reply elections/steering/2023/election.yaml]: (I do have a voters.yaml but if the specific empty list format Elekto would need is not what the example file comes with, perhaps we should adjust the example?)

**Comment by bridgetkromhout** [comment]
As listed in https://github.com/kubernetes/community/tree/master/elections/steering/documentation#create-a-timeline:

_Once EOs have a draft timeline, they will submit it to Steering to approve, either as an issue in k/steering, or as a PR that creates the README for the election. Asking for review on Slack is recommended. Once a majority of Steering members have approved, the timeline is final._

@dims @kaslin and I are requesting approval of the timeline.

_At the same time you have Steering approve the timeline, they should also approve the voting eligibility criteria. As such, you should propose using the same criteria as last year; Steering will make any changes that they need to._

Please suggest changes to the voting eligibility as needed - thanks!

**Comment by bridgetkromhout** [comment]
/hold

I think we need to make sure this doesn't merge before the majority of Steering have approved it.

**Review by cpanato — APPROVED** [review]
thanks
/lgtm

File: elections/steering/2023/election_desc.md
Comment (cblecker) [inline elections/steering/2023/election_desc.md:1]: This seems confusing.. isn't this just the 2023 election?

File: elections/steering/2023/election_desc.md
Comment (tpepper) [inline elections/steering/2023/election_desc.md:3]: Nit here for a separate update to the template (along with the typos fixed in the downstream copy):  Instead of "fall" with its Northern Hemisphere bias implied, can we say something like:  "As is now customary, the third calendar quarter is...

File: elections/steering/2023/election_desc.md
Comment (bridgetkromhout) [reply elections/steering/2023/election_desc.md:1]: Hi, @cblecker - You are correct that it's the 2023 election, and probably it could be called the 2023 Steering Committee - although this "year" runs through part of 2023 and part of 2024.

I based this update on what I saw in [2022](https://github.com/kubernetes/community/blame/master/elections/steering/2022/election_desc.md#L1) and [2021](https://github.com/kubernetes/community/blame/master/elections/steering/2021/election_desc.md#L1) but if you think it should say "2023" instead of "2023-2024" I can break with the convention of the last two elections.

**Comment by bridgetkromhout** [comment]
Steering committee approval of the election timeline and voter eligibility:
- [ ] @BenTheElder
- [ ] @cblecker
- [ ] @cpanato 
- [ ] @justaugustus 
- [ ] @mrbobbytables 
- [ ] @palnabarun 
- [ ] @tpepper

**Review by palnabarun — APPROVED** [review]
/lgtm
/approve
(with steering hat)

File: elections/steering/2023/election.yaml
Comment (jberkus) [reply elections/steering/2023/election.yaml]: No no no, you misunderstand me.  Having the blank election up is a *good* thing.

**Review by BenTheElder — COMMENTED** [review]
/lgtm
/approve
Thank you!

File: elections/steering/2023/election_desc.md
Comment (cblecker) [reply elections/steering/2023/election_desc.md:1]: It's probably just nit picky, but yeah, I think we should talk about this being the 2023 election as even the term that folks will be elected for is 2 years.. so you're just just electing for a 12 month period.

**Comment by bridgetkromhout** [comment]
I believe we have addressed all feedback and this is ready to merge once it gets a final LGTM.

/hold cancel

**Comment by dims** [comment]
/lgtm
/approve

off we go! 

/pony

**Pull request merged**
